docs: changes to docs (#1338)
graphcommerce-org/graphcommerce: docs/getting-started/create.md

To order to be able to connect your GraphCommerce app to your Magento backend
and/or GraphCMS project, you'll need:

> - Magento 2.4.3 - Clean install, or a production or a development environment

Author: ```suggestion
- Magento 2.4.3 - Clean install, or a production or a development environment
+ Magento 2.4.3 - Clean install, a production or a development environment
```

> - Magento 2.4.3 - Clean install, or a production or a development environment

Author: ```suggestion
- Magento 2.4.3 - Clean install, a production or a development environment
```

Visit the development environment running at http://localhost:3000  
Visit the GraphQL Playground running at http://localhost:3000/api/graphql

> No success? Refer to [common build errors](../framework/troubleshooting.md) or

> > [ask a question on Slack ↗](https://join.slack.com/t/graphcommerce/shared_invite/zt-11rmgq1ad-F~0daNtKcSvtcC4eQRzjeQ)

Author: ```suggestion
> ask your question in the public [Slack community ↗](https://join.slack.com/t/graphcommerce/shared_invite/zt-11rmgq1ad-F~0daNtKcSvtcC4eQRzjeQ) chat
```
